lib/logflare/single_tenant.ex

```diff
@@ -163,4 +169,37 @@ defmodule Logflare.SingleTenant do
       false
     end
   end
+
+  def supabase_mode? do
+    if Application.get_env(:logflare, :supabase_mode) do
+      true
+    else
+      false
+    end
+  end
```

Comment:
```suggestion
  def supabase_mode?, do: !! Application.get_env(:logflare, :supabase_mode)
```

Maybe a bit to similar to JS but does cut lines of code 😅